cni/network/plugin/main.go

```diff
@@ -15,9 +15,10 @@ import (
 	"github.com/Azure/azure-container-networking/aitelemetry"
 	"github.com/Azure/azure-container-networking/cni"
 	"github.com/Azure/azure-container-networking/cni/api"
-	"github.com/Azure/azure-container-networking/cni/log"
+	zaplog "github.com/Azure/azure-container-networking/cni/log"
```

Comment:
is this change needed? i'm not sure why the import alias is prefixed with `zap` (seems like an implementation detail). less changed lines if you just alias the new conflicting import?

Reply:
Hi Ramiro, because we want to rename the import packet since they are all using "log." to avoid conflicts